wolfcrypt/src/rsa.c

```diff
@@ -4035,7 +4035,6 @@ int wc_RsaPSS_CheckPadding_ex2(const byte* in, word32 inSz, byte* sig,
         }
     }
 
-#ifdef WOLFSSL_PSS_LONG_SALT
```

Comment:
I don't think removing this macro for long salt is the right solution. Instead check size and fail in the `#else` case.